crates/control-plane-api/src/server/error.rs

```diff
@@ -129,20 +142,38 @@ impl ApiError {
     }
 }
 
+impl async_graphql::ErrorExtensions for ApiError {
+    fn extend(&self) -> async_graphql::Error {
+        let err = async_graphql::Error::new(self.error.to_string());
```

Comment:
use format!("{:#}") so we include full error chains.

to_string() drops all but the highest-level error context.

Any other places we're doing this?